.github/workflows/smoke.yml

```diff
@@ -128,6 +129,211 @@ jobs:
         curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sudo sh -s -- -b /usr/bin v1.51.2
         sudo -E make smoke-only
 
+  benchmark-oci:
+    runs-on: ubuntu-latest
+    needs: [contrib-build, nydus-build]
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Download Nydus
+      uses: actions/download-artifact@master
+      with:
+        name: nydus-artifact
+        path: target/release
+    - name: Download Nydusify
+      uses: actions/download-artifact@master
+      with:
+        name: nydusify-artifact
+        path: contrib/nydusify/cmd
+    - name: Prepare Nerdctl Environment
+      run: |
+        sudo install -D -m 755 contrib/nydusify/cmd/nydusify /usr/local/bin
+        sudo install -D -m 755 target/release/nydusd target/release/nydus-image /usr/local/bin
+        sudo wget https://github.com/containerd/nerdctl/releases/download/v1.3.0/nerdctl-1.3.0-linux-amd64.tar.gz
+        sudo tar -xzvf nerdctl-1.3.0-linux-amd64.tar.gz -C /usr/local/bin
+        sudo mkdir -p /opt/cni/bin
+        sudo wget https://github.com/containernetworking/plugins/releases/download/v1.2.0/cni-plugins-linux-amd64-v1.2.0.tgz 
+        sudo tar -xzvf cni-plugins-linux-amd64-v1.2.0.tgz -C /opt/cni/bin
+        sudo install -D misc/benchmark/cni_bridge.conf /etc/cni/net.d/bridge.conf
+    - name: Prepare OCI Registry Environment
+      run: |
+        sudo docker run -d --restart=always -p 5000:5000 --name registry registry
+        sudo docker pull ${{env.IMAGE}} && docker tag ${{env.IMAGE}}:latest localhost:5000/${{env.IMAGE}}:latest
+        sudo docker push localhost:5000/${{env.IMAGE}}:latest
+        git clone https://github.com/magnific0/wondershaper.git
+        sudo install -D -m 755 wondershaper/wondershaper /usr/local/bin
+    - name: BenchMark Test
+      run: |
+        cd misc/benchmark
+        sudo python3 benchmark.py --mode oci
+    - name: Save Test Result
+      uses: actions/upload-artifact@v3
+      with:
+        name: benchmark-oci
+        path: misc/benchmark/${{env.IMAGE}}.csv
```

Comment:
Should be `misc/benchmark-oci/${{env.IMAGE}}.csv` ?

Reply:
Each test generates ${{env.IMAGE}}.csv in the current directory, the upload to the artifac. So  misc/benchmark-oci/${{env.IMAGE}}.csv is not better for smoke test. How about change to misc/benchmark/oci/${{env.IMAGE}}.csv? It will work better in [https://github.com/dragonflyoss/image-service/pull/1219#issuecomment-1510574406](https://github.com/dragonflyoss/image-service/pull/1219#issuecomment-1510574406)